samples/sample_full_pbr.cpp

```diff
@@ -281,7 +281,9 @@ static void setup(Engine* engine, View* view, Scene* scene) {
             vec2 uvDy = dFdy(uv0);
 
             mat3 tangentFromWorld = transpose(getWorldTangentFrame());
-            vec3 v = tangentFromWorld * getWorldViewVector();
+            vec3 tangentCameraPosition = tangentFromWorld * getWorldCameraPosition();
+            vec3 tangentFragPosition = tangentFromWorld * getWorldPosition();
+            vec3 v = normalize(tangentCameraPosition - tangentFragPosition);
```

Comment:
This part is tentative. If Google accepts our changes, I'll revert this, otherwise it stays in the PR

Reply:
The upstream PR got merged, so I decided to keep the view dir. fixes in this sample project (and reverted the revert in c1d2a6c3b3a9d1ea35482e5ac9d7ce51e412cd04)